mantidproject/mantid
# Fix SANS Export Table in Workbench

**Description of work.**
ISIS SANS has an `Export Table` button, which exports data in the current table as a csv file. This was broken in workbench as qt5 filename widgets return a tuple of `(file name, file extension)`, whereas qt returns the filename as a string only.
I also changed some logger levels I thought were incorrect.

**To test:**
1. Open Workbench
2. Interfaces -> SANS -> ISIS SANS
3. In the table, type any number in the leftmost column
4. Click the `Export Table` button and select a file location/filename and save it
5. Check that this file now exists

Fixes #25438 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
